mobile/lib/common/domain/model.dart

```diff
@@ -11,6 +11,11 @@ class Amount {
     _sats = Decimal.fromInt(sats);
   }
 
+  // TODO: this is bad for precision
+  Amount.fromBtc(double btc) {
```

Comment:
Note: this is really not good, we should not be using Amount to represent USD. I just did it this way because it's what's being used everywhere else in the code. We should aim to address the model issue after LoL/when time crunch is off